tslearn/neighbors.py

```diff
@@ -4,24 +4,23 @@
 """
 
 import numpy
+from sklearn import neighbors
 from sklearn.neighbors import KNeighborsClassifier, NearestNeighbors
 from sklearn.neighbors.base import KNeighborsMixin
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
 from scipy.spatial.distance import cdist as scipy_cdist
 
 from tslearn.metrics import cdist_dtw
-from tslearn.utils import to_time_series_dataset, to_sklearn_dataset
+from tslearn.utils import (to_time_series_dataset, to_sklearn_dataset,
+                           _check_dims)
+
+neighbors.VALID_METRICS['brute'].append('dtw')
```

Comment:
This is probably one of the additions I'm not very satisfied with. This should probably be done somewhere on a higher top-level? 

I also removed https://github.com/rtavenar/tslearn/blob/master/tslearn/neighbors.py#L18 as the same thing was being done in https://github.com/rtavenar/tslearn/blob/master/tslearn/neighbors.py#L25, but perhaps it could be added again so that everything becomes more clear?